src/openai/lib/streaming/chat/_completions.py

```diff
@@ -236,9 +236,15 @@ def current_completion_snapshot(self) -> ParsedChatCompletionSnapshot:
 
     async def __stream__(self) -> AsyncIterator[ChatCompletionStreamEvent[ResponseFormatT]]:
         async for sse_event in self._raw_stream:
-            if not _is_valid_chat_completion_chunk_weak(sse_event):
+            valid = _is_valid_chat_completion_chunk_weak(sse_event)
+            if valid is False:
                 continue
```

Comment:
I don't think this makes sense, when can this function return anything other than a `bool`?